microcosm_postgres/tests/test_employee_store.py

```diff
@@ -279,3 +279,25 @@ def test_search_first(self):
         assert_that(retrieved_real_employee.last, is_(equal_to("Doe")))
         retrieved_fake_employee = self.employee_store.search_first(first="Tarzan")
         assert_that(retrieved_fake_employee, is_(equal_to(None)))
+
+    def test_search_ofer_by(self):
+        """
+        Should be sorted by created_at in descending order.
```

Comment:
Descending order means new updates with get in the first page, thus making pagination non-stable. Is this what we want?

Reply:
I can see both directions. Descending is typically much more useful, but you're right that it's a poor default.

Maybe:
 - Default to ascending
 - Add some hook to the `_order_by` kwargs to invert